lighthouse-viewer/types/viewer.d.ts

```diff
@@ -29,7 +26,9 @@ declare global {
     // Inserted by viewer build.
     LH_CURRENT_VERSION: string;
   }
+  
+  // Expose global types in LH namespace.
+  module LH {
+    export import ReportResult = _ReportResult;
```

Comment:
Oh interesting I wouldn't have expected `export import` to be needed since we're reassigning.

does this differ from `export type ReportResult = _ReportResult;` or because it's a module we can't do that and `export import` is the only valid way?

Reply:
> Oh interesting I wouldn't have expected `export import` to be needed since we're reassigning.
> 
> does this differ from `export type ReportResult = _ReportResult;` or because it's a module we can't do that and `export import` is the only valid way?

yeah, you get the interface but not the module, so `LH.ReportResult.AuditRef` won't resolve